change/@microsoft-fast-foundation-bebb95bc-b4fb-48e2-8371-85e34ce9ee40.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "remove activeindicator from tabs component",
+  "packageName": "@microsoft/fast-foundation",
```

Comment:
Should these be 'prerelease' since this change affects distributed package?

Reply:
Oh this should for sure. I may have incidentally just hit enter when the other file got prompted. Good catch.

Reply:
resolved!